app/build.gradle.kts

```diff
@@ -1,4 +1,7 @@
+import com.android.build.gradle.internal.cxx.configure.gradleLocalProperties
```

Comment:
This `internal` here poked my eye a bit, but _I'm sure it's fine_.

Reply:
It's the best we got... :D